### PR TITLE
ok-to-test changes for components-contrib

### PR DESCRIPTION
**Pull request by tanvigour**
# Description

_Please explain the changes you've made_

## Issue reference #1070 

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly : verified the ok-to-test functionality in forked repo PR
* [ ] Created/updated tests
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

File: bindings/azure/eventhubs/eventhubs.go [hunk 1]
```diff
@@ -198,10 +198,11 @@ func (a *AzureEventHubs) RegisterPartitionedEventProcessor(handler func(*binding
 		return err
 	}
 
-	callback := func(c context.Context, event *eventhub.Event) error {
+	callback := func(c context.Context, event *eventhub.Event, bind *bindings.InvokeResponse) error {
```
Comment (CodeMonkeyLeet) [inline bindings/azure/eventhubs/eventhubs.go:201]: Is this change supposed to be part of this PR?

Comment (tanvigour) [reply bindings/azure/eventhubs/eventhubs.go:201]: no it's not, I am not sure how it even got attacked to this

Comment (tanvigour) [reply bindings/azure/eventhubs/eventhubs.go]: Thanks for catching that. Looks like my fork had that commit and probably just got pushed along with ok-to-test commit, just reverted it.

File: .github/workflows/dapr-bot.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+                  // Fire repository_dispatch event to trigger conformance test
+                  await github.repos.createDispatchEvent({
+                    owner: issue.owner,
+                    repo: issue.repo,
+                    event_type: "conformance-test",
+                    client_payload: testPayload,
+                  });
+      
+                  // Fire repository_dispatch event to trigger e2e test
```
Comment (CodeMonkeyLeet) [inline .github/workflows/dapr-bot.yml:45]: There's no `e2e-test` in the components-contrib repo yet ... is this leftover from the copy from dapr/dapr?

File: .github/workflows/dapr-bot.yml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+                  await github.repos.createDispatchEvent({
+                    owner: issue.owner,
+                    repo: issue.repo,
+                    event_type: "e2e-test",
+                    client_payload: testPayload,
+                  });
+                  
+                  console.log(`Trigger conformance and E2E test for ${JSON.stringify(testPayload)}`);
+              }
+            }
```
Comment (CodeMonkeyLeet) [inline .github/workflows/dapr-bot.yml:55]: Nit: newline at the end-of-file for Linux

**Review by CodeMonkeyLeet — COMMENTED** [review]
Looks generally okay except for the extra dispatch.

You should also merge from upstream master and resolve the conflict in your branch ... https://github.com/dapr/components-contrib/pull/1058 introduced the dapr-bot.yml already and your changes need to be reconciled with those which pulled in support for the `/assign` command.

File: .github/workflows/dapr-bot.yml (hunk 2 above)
Comment (tanvigour) [reply .github/workflows/dapr-bot.yml:46]: this is leftover from dapr/dapr, but I wonder if we should just leave it there for when we do have e2e tests?

Comment (tanvigour) [reply .github/workflows/dapr-bot.yml]: removed it

**Review by CodeMonkeyLeet — APPROVED** [review]
LGTM!

**Comment by berndverst** [comment]
@artursouza please review

**Review by artursouza — APPROVED** [review]

**Pull request merged**